ChangeLog

```diff
@@ -1,5 +1,18 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS x.x.x released xxxx-xx-xx
+
+Changes
+   * Added config.h option MBEDTLS_NO_UDBL_DIVISION, to prevent the use of
+     64-bit division.
+   * Added config.h option MBEDTLS_TYPE_UDBL to allow configuring the
```

Comment:
I'd suggest to rename `MBEDTLS_NO_UDBL_DIVISION` to `MBEDTLS_NO_UDBL` and document it in the ChangeLog as "to prevent the use of double-width integers". This would be in accordance with the documentation "Double-width integers (e.g. 128-bit in 64-bit architectures) can be disabled by defining MBEDTLS_NO_UDBL_DIVISION." in the comment below. The documentation in `config.h` would need to be adapted accordingly.